x-pack/plugin/core/src/test/java/org/elasticsearch/license/AbstractClusterStateLicenseServiceTestCase.java

```diff
@@ -32,9 +32,9 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-public abstract class AbstractLicenseServiceTestCase extends ESTestCase {
+public abstract class AbstractClusterStateLicenseServiceTestCase extends ESTestCase {
 
-    protected ClusterStateLicenseService clusterStateLicenseService;
+    protected LicenseService.MutableLicenseService licenseService;
```

Comment:
Nit: I think we can keep the existing `ClusterStateLicenseService` variable given this class of tests are specifically about this particular implementation. It helps saves some boilerplate in subclasses.